analysis_configs/t_channel_lost_lepton_control_region.py

```diff
@@ -84,6 +84,9 @@ def process(events, cut_flow, year, primary_dataset="", pn_tagger=False, **kwarg
     events = events[events.MET > 200]
     skimmer_utils.update_cut_flow(cut_flow, "METGt200GeV", events)
 
+    # Phi spike filter
```

Comment:
Don't we want to apply this together with the MET filter right after the standard MET filters? That would make more sense to have all the MET filters (standard and custom) together, no? This is what is done in the s-channel AN.
Maybe the easiest to keep a cutflow table which resembles the one we currently have would be to move the standard MET filter cut in between the MET cut and the phi spike filter? Because as Kevin pointed out, the reason why the filters are needed is because we required MET, so it would make sense to have those filters after the MET cut!